index.html

```diff
@@ -1,29 +1,195 @@
 <!DOCTYPE html>
-<html lang="en-ca">
-  <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.cdnfonts.com/css/anurati" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.typekit.net/xse6ume.css">
+    <title>Assignment 2</title>
+</head>
+<body>
+    <!-- Header -->
+    <div class="header">
+        <h1>OUR ECOLOGY</h1>
+        <p><a href="#">Home</a></p>
+        <p><a href="#mission">Mission</a></p>
+        <p><a href="#">Community</a></p>
+        <p><a href="#news">News</a></p>
+        <p><a href="#">About Us</a></p>
+        <p><a href="#contact">Contact</a></p>
+        <p><a href="#donate">Donate</a></p>
+        <!-- Search bar text and icon -->
+            <a><input type="text" placeholder="Search"></a>
+                <img src="search.png" alt="Search icon" width="30">
+    </div>
 
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Specifies the document mode for Internet Explorer -->
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Hero -->
+    <div class="hero">
+        <h1>MAKE A CHANGE TO THE <br>
+            ENVIRONMENT TODAY.</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br>
+            tempor incididunt ut labore et dolore magna aliqua. Et ligula ullamcorper <br>
```

Comment:
Your `main` element should start here and it should wrap all of your main content (all of the content that isn't part of the header or footer).